src/java/org/apache/cassandra/db/filter/RowFilter.java

```diff
@@ -149,7 +149,8 @@ public boolean isSatisfiedBy(TableMetadata metadata, DecoratedKey partitionKey,
         if (purged == null)
             return root.isEmpty();
 
-        return root.isSatisfiedBy(metadata, partitionKey, purged);
+        boolean satisfied = root.isSatisfiedBy(metadata, partitionKey, purged);
```

Comment:
I'm assuming that this was left over from debugging. I'd prefer that the change was reverted so we make minimal changes to the core db code.